tensorflow_addons/metrics/hamming.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements Hamming distance and loss."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from tensorflow.keras.metrics import Metric
+from tensorflow_addons.metrics.utils import MeanMetricWrapper
+
+def hamming_distance(actuals, predictions):
```

Comment:
Make the params for `hamming_distance` and `hamming_loss` the same. Use `y_true`, `y_pred` or `actuals`, `predictions.`

Reply:
@SSaishruthi update this